modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/Helper.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Godot.SourceGenerators
+{
+    public static class Helper
+    {
+        [Conditional("DEBUG")]
+        public static void ThrowWhenNull([NotNull] object? value)
```

Comment:
I would name it `ThrowIfNull` to match .NET throw helpers (e.g.: [`ArgumentNullException.ThrowIfNull`](https://github.com/dotnet/runtime/blob/205ef031e0fe5152dede0bd9f99d0f6f9e7f1e45/src/libraries/System.Private.CoreLib/src/System/ArgumentNullException.cs#L62))